pytorch/buildspec.yml

```diff
@@ -21,15 +21,16 @@ context:
       target: braket_container.py
 
 images:
-  BuildPytorchCPUJobsPy3DockerImage:
+  BuildPytorchGPUJobsPy3DockerImage:
     <<: *JOBS_REPOSITORY
-    build: &PYTORCH_CPU_JOBS_PY3 false
-    image_size_baseline: 4899
-    device_type: &DEVICE_TYPE cpu
+    build: &PYTORCH_GPU_JOBS_PY3 false
+    image_size_baseline: 16000
```

Comment:
That's quite a bit bigger.

Reply:
cuquantum is around 6 gigs to pull in

Reply:
Is it needed in the TF container?

Reply:
Still twice the size, even without cuquantum.

Reply:
It is needed